tower-http-util/Cargo.toml

```diff
@@ -21,10 +21,8 @@ Utilities for working with HTTP services.
 """
 
 [dependencies]
-futures = "0.1.25"
-http = "0.1.16"
-http-body = "0.1"
+http = "0.2"
+http-body = "0.3"
 http-connection = "0.1.0"
-tokio-buf = "0.1.0"
-tokio-io = "0.1"
-tower-service = "0.2.0"
+tokio = "0.2"
```

Comment:
```suggestion
tokio = { version = "0.2", features = ["io"] }
```

Reply:
`tokio` v0.2 does not have have a feature named `io`. Also, ` tower-http-util` only uses `AsyncRead` and `AsyncWrite`, which do not depend on any Cargo features.

Reply:
You're right!